src/Button/__tests__/defaultButton.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render, cleanup } from '@testing-library/react';
+import { DefaultButton } from '../index';
+
+afterEach(cleanup);
+
+describe('Test that DefaultButton will render without crashing', () => {
```

Comment:
You don't need to test that it will render without crashing because react will test for that already.  Instead, you should test that the button is in the document or on the DOM and the different interactions the user can have with the different states of the button.

ie: Does it trigger a click if the user clicks it?  Does it allow the user to click it if it's disabled? Etc...